Desktop/main.cpp

```diff
@@ -148,6 +150,8 @@ void parseArguments(int argc, char *argv[], std::string & filePath, bool & newDa
 		else if(args[arg] == "--safeGraphics")					safeGraphics			= true;
 		else if(args[arg] == "--newData")						newData					= true;
 #ifdef _WIN32
+		else if(args[arg] == "--sandbox")			{			containerSettingForced	= true;		container = true; }
+		else if(args[arg] == "--noSandbox")			{			containerSettingForced	= true;		container = false; }
```

Comment:
Need to be carefully here that qtwebengine does also has a arguments of --sandbox, I'm not sure if this will effect on that.

Reply:
ah,sorry qtwebengine called it "--no-sandbox".

Reply:
Yes.